backend/src/main/java/pl/edu/agh/awiteks_backend/models/ForumThread.java

```diff
@@ -22,22 +23,28 @@ public class ForumThread {
     @ManyToOne(fetch = FetchType.LAZY, optional = false)
     @JoinColumn(name = "user_id", nullable = false)
     @Schema(required = true)
-    private User creator;
+    private User user;
 
     @OneToMany(mappedBy = "thread", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
     @Schema(required = true)
     private List<ForumPost> forumPosts;
 
-    public ForumThread(Integer id, String name, User user) {
-        this.id = id;
-        this.creator = user;
+    @ManyToMany(mappedBy = "followedThreads", cascade = CascadeType.ALL)
+    @Schema(required = true)
+    private List<User> followingUsers;
+
+    @Schema(required = false)
+    private final LocalDateTime creationTime = LocalDateTime.now();
```

Comment:
Skoro dodaliście nowe pola do modelu, to należy zaktualizować plik `gen-test-data.sql`, w którym jest przykład bazy danych, aby można było z tego korzystać